learn/react-native/mobile-build-manual.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "React Native - Mobile Manual Build"
+id: "mobile-build-manual"
+sidebar_label: "Manual Build"
+---
+---
+
+You can generate the WaveMaker studio project in the local environment where additional customization and debugging of project code can be performed in the manual build process using React Native Command Line Interface (CLI). The React Native CLI commands are also used to preview the application.
```

Comment:
Still not clear. 

Are you saying local manual build allows developers to build app that is easier to debug using ADB? then mention that clearly.

Reply:
In this document we are only trying to discuss the manual build process to run commands to generate the artifacts where ADB has no role. I have changed the introduction and removed the information on generating the project, customization and debugging, which fall under previewing where ADB is used.